Source/WebCore/style/StyleInvalidator.cpp

```diff
@@ -297,8 +297,10 @@ void Invalidator::invalidateStyleWithMatchElement(Element& element, MatchElement
             invalidateIfNeeded(*sibling, nullptr);
         break;
     case MatchElement::AnySibling:
-        for (auto& parentChild : childrenOfType<Element>(*element.parentNode()))
-            invalidateIfNeeded(parentChild, nullptr);
+        if (CheckedPtr parentNode = element.parentNode()) {
```

Comment:
I don't think this fix makes sense a we talked about it in slack. We shouldn't be calling this function with a parentless element.